GVRf/Framework/jni/objects/scene_object.cpp

```diff
@@ -250,31 +250,217 @@ bool SceneObject::isColliding(SceneObject *scene_object) {
 }
 
 void SceneObject::dirtyBoundingVolume() {
```

Comment:
What do you think about renaming this function to reflect that the value could also be a HBV?

It might not be straightforward for developers to differentiate between this call and renderdata->mesh->boundingvolume() without looking at the source code.